backend/metamodel/syson-sysml-metamodel-edit/src/main/java/org/eclipse/syson/sysml/provider/NamespaceItemProvider.java

```diff
@@ -183,6 +185,37 @@ public Object getImage(Object object) {
         return this.overlayImage(object, this.getResourceLocator().getImage("full/obj16/Namespace.svg"));
     }
 
+    /**
+     * {@inheritDoc}.
+     *
+     * @generated NOT
+     */
+    @Override
+    protected Object overlayImage(Object object, Object image) {
+        Object superResult = super.overlayImage(object, image);
+        if (object instanceof Namespace namespace && namespace.getOwningMembership() != null) {
+            List<Object> images = new ArrayList<>(2);
+            images.add(image);
```

Comment:
`image` and `superResult` seems to always return the same result for now as we don't have any other icons overlay in the code.
May be this code could be replace by:
```
if (superResult != null) {
                images.add(superResult);
} else {
                images.add(image);
}
```
What do you think about?